ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_kvutils.proto

```diff
@@ -199,16 +199,16 @@ message DamlTransactionRejectionEntry {
     // Rejections used by both participant.state v1 and v2 API.
     InvalidLedgerTime invalid_ledger_time = 9;
     SubmitterCannotActViaParticipant submitter_cannot_act_via_participant = 8;
+    Duplicate duplicate_command = 6;
```

Comment:
The field order in this message is quite confusing. Shall we reorder these 3 ascending according to their ID to have consistency per group of fields?

Reply:
Done, makes sense as it was weird to have them in reverse order